assignment.js

```diff
@@ -57,17 +64,28 @@ Use the following test cases to confirm your program meets the success criteria
 2. Given the following:
     problem1Numbers = [9, 18, 62, 3, 17]
 
-   Result: 0
+   Result: 21.8
 
 ************/
 
 console.log('Problem 2:')
 
 // Add your code below this line
 
+let numbers = [9, 18, 62, 3, 17]
+let average = 0
+let indexTotal = 0
+
+numbers.forEach(function(element, index) {
```

Comment:
@kyleligman in the future you can leave out the `index` parameter if you're not going to use it during your `forEach()`

Reply:
As an alternative to `.forEach()`, `.reduce()` would have also been a good candidate here to calculate the sum all the numbers in the array: 

`const total = numbers.reduce((sum, number) => sum + number)`